app/views/layouts/application.html.haml

```diff
@@ -1,6 +1,7 @@
 !!!
 %html{ lang: "en-US" }
   %head
+    = render "layouts/meta_tags"
```

Comment:
`action_cable_meta_tag` must load before the JS. Otherwise, we wouldn't get the correct URL for the cable consumer to connect to.

Reply:
Nice spot!